forte/casscf/mcscf_2step.cc

```diff
@@ -441,23 +460,17 @@ bool MCSCF_2STEP::is_single_reference() {
     return false;
 }
 
-double
-MCSCF_2STEP::diagonalize_hamiltonian(std::shared_ptr<ActiveSpaceSolver>& as_solver,
-                                     std::shared_ptr<ActiveSpaceIntegrals> fci_ints,
-                                     const std::tuple<int, double, double, bool, bool>& params) {
-    auto state_nroots_map = to_state_nroots_map(state_weights_map_);
-    as_solver = make_active_space_solver(ci_type_, state_nroots_map, scf_info_, mo_space_info_,
-                                         fci_ints, options_);
-
-    int print;
-    double e_conv, r_conv;
-    bool read_wfn_guess, dump_wfn;
-    std::tie(print, e_conv, r_conv, read_wfn_guess, dump_wfn) = params;
+double MCSCF_2STEP::diagonalize_hamiltonian(std::shared_ptr<ActiveSpaceSolver>& as_solver,
```

Comment:
There is a bug at line 419 `auto nclosed_electrons = 2 * mo_space_info_->size("INACTIVE_DOCC");` where the 2 should be removed.